ssc/cmod_grid.cpp

```diff
@@ -128,20 +128,22 @@ void cm_grid::exec() throw (general_error)
         load_year_one = as_vector_double("load");
     }
 
+
+    /* Does nothing since "load_escalation" not an input to this compute module
```

Comment:
I like this comment, but don't see the reason to leave the code commented out. Why not just remove it?